sources/packages/api/src/route-controllers/application/application.controller.service.ts

```diff
@@ -203,4 +209,78 @@ export class ApplicationControllerService {
       );
     }
   }
+
+  /**
+   * Process application income verification details.
+   * @param CRAIncomeVerification CRA income verification data.
+   * @return processed object.
+   */
+
+  processApplicationIncomeVerificationDetails(
+    CRAIncomeVerification: CRAIncomeVerification[],
+  ): ApplicationIncomeVerification {
+    const incomeVerificationDetails = {} as ApplicationIncomeVerification;
+    const studentIncomeVerification = CRAIncomeVerification.filter(
+      (incomeVerification) => !incomeVerification.supportingUser,
+    );
+    const supportingUserIncomeVerification = CRAIncomeVerification.filter(
+      (incomeVerification) => incomeVerification.supportingUser,
+    );
+    studentIncomeVerification.forEach((incomeVerification) => {
```

Comment:
I believe that there is no need here to do a for each when we expected only one record. We can access the first item in the array only.
```ts
const [studentIncomeVerification] = studentIncomeVerifications;
```